pkg/cmd/image/list.go

```diff
@@ -121,21 +121,6 @@ func filterByReference(imageList []images.Image, filters []string) ([]images.Ima
 	return filteredImageList, nil
 }
 
-func filterDangling(imageList []images.Image, dangling bool) []images.Image {
```

Comment:
can we keep the func here? Since there are other image filter funcs here (filterByReference, filterByLabel).

Reply:
Sure. Done.